packages/loopring_v3/contracts/amm/libamm/AmmStatus.sol

```diff
@@ -37,20 +37,21 @@ library AmmStatus
         require(config.tokens.length == config.weights.length, "INVALID_DATA");
         require(config.tokens.length >= 2, "INVALID_DATA");
         require(config.exchange != address(0), "INVALID_EXCHANGE");
-        require(config.accountID != 0, "INVALID_ACCOUNT_ID");
-
         require(S._totalSupply == 0, "POOL_IN_USE");
 
+        IExchangeV3 exchange = IExchangeV3(config.exchange);
+
         S.feeBips = config.feeBips;
         S.domainSeparator = EIP712.hash(EIP712.Domain(config.poolName, "1.0.0", address(this)));
-
         S.poolName = config.poolName;
         S.symbol = config.tokenSymbol;
+        S.sharedConfig = IAmmSharedConfig(config.sharedConfig);
```

Comment:
I propose to re-initialize these state variables again in case some values become different.